src/plugins/intel_cpu/tests/unit/shape_inference_test/custom_shape_infer/adaptive_avg_pool.cpp

```diff
@@ -0,0 +1,77 @@
+// Copyright (C) 2018-2023 Intel Corporation
+// SPDX-License-Identifier: Apache-2.0
+//
+
+#include <gtest/gtest.h>
+#include "common_test_utils/test_assertions.hpp"
+#include "custom_shape_infer.hpp"
+#include <ngraph/opsets/opset8.hpp>
```

Comment:
Cosmetic: Use `openvino` includes instead `ngraph`. Use `""` for openvino includes..
If changed check other new added files.

Reply:
fixed